### PR TITLE
Create src directory

### PR DESCRIPTION
**Pull request by EPashkin**

### CODE REVIEW

**Comment by EPashkin** [comment]
Its good time to add `Gir_` prefix (or `gir_`) to tomls, if you allow.

File: tests/sys/test.sh [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-LIBS="glib gobject gio pango"
 export RUST_LOG="gir=warn"
 
 cd "`dirname $0`"
 cargo build --release || exit 1
 
-for LIB in $LIBS; do
-	mkdir -p ${LIB}-sys/src
-	$GIR -c ${LIB}.toml || exit 2
+for TOML in `ls *.toml`; do
```
Comment (gkoz) [inline tests/sys/test.sh]: `for TOML in *.toml` should work too

File: src/codegen/sys/lib_.rs [hunk 2]
```diff
@@ -18,9 +19,12 @@ pub fn generate(env: &Env) {
 
     let path =  PathBuf::from(&env.config.target_path)
         .join(file_name_sys("lib"));
-    println!("Generating file {:?}", path);
 
-    save_to_file(path, &mut |w| generate_lib(w, env));
+    let parent = path.parent().unwrap();
+    fs::create_dir(parent).ok();
```
Comment (gkoz) [inline src/codegen/sys/lib_.rs]: Haha, this is a novel way of ignoring errors. `let _ = ...` might be more obvious.

**Comment by EPashkin** [comment]
Really works. I tried `for TOML in "*.toml"` and it not work.

**Comment by gkoz** [comment]
Yes, renaming is fine. Consider `-` instead of `_` if it's not against your taste ;)

File: src/codegen/sys/lib_.rs (hunk 2 above)
Comment (EPashkin) [inline src/codegen/sys/lib_.rs]: You yourself have taught me that

**Comment by EPashkin** [comment]
Only problem with `-` it mixed symbols in `gir-gdk_pixbuf'.

**Comment by EPashkin** [comment]
But we already have directory like `gio-sys` so be `gir-gdk-pixbuf.toml`

**Comment by gkoz** [comment]
Thanks!
@homu r+

**Pull request merged**